### ISSUE TITLE
Add manual page for program

### ISSUE THREAD

**Issue by ExcaliburZero** [ExcaliburZero/jekyll-helper#34]
Currently the program does not have a manual page that can be shown via `man jekyll-helper` in order to provide quick information on the program. Thus, such as page should be added.

**Comment by ExcaliburZero** [comment]
These pages have some information on creating manual pages:
- [Creating Your Own MAN Page Version 1.0](http://www.linuxhowtos.org/System/creatingman.htm)
- [HowTo: Linux / UNIX Create a Manpage](http://www.cyberciti.biz/faq/linux-unix-creating-a-manpage/)

**Comment by ExcaliburZero** [comment]
It looks like the file for a manual pages is written something like the following.

```
.TH foo 1 "14 May 1999" "version 1.0"
.SH NAME
foo - my own text editor
.SH SYNOPSIS
.B [-d] [-e] [-f
.I filename
.B ]
```

**Comment by ExcaliburZero** [comment]
It might be a good idea to see how manual pages are implemented in other programs such as [Trimage](https://github.com/Kilian/Trimage/blob/master/doc/trimage.1).

**Comment by ExcaliburZero** [comment]
It looks like Trimage implements its manual page through the following:
- [Commit: Add man page, by Kyrill Detinov](https://github.com/Kilian/Trimage/commit/f17cbf980ac2f78966beb0c7511aef3705df75d5)
- [Commit: add doc page, add it's author to the index.html,. fixes issue #17](https://github.com/Kilian/Trimage/commit/8a8ffa66ef213453ea5da413c72e79273f05b0f0)

**Comment by ExcaliburZero** [comment]
The manual file should probably be licensed under the [GNU Free Documentation License version 3](http://www.gnu.org/licenses/fdl-1.3.en.html).
